src/components/graphic.js

```diff
@@ -1,24 +1,20 @@
 import React from "react";
-import { GridContainer, Grid } from "@trussworks/react-uswds";
+import { Grid } from "@trussworks/react-uswds";
 
 const Graphic = ({ graphics }) => {
   return (
-    <GridContainer>
-      <Grid row>
-        {graphics.map((graphic) => {
+    <Grid row>
+      {graphics &&
+        graphics.map((graphic) => {
           return (
-            <Grid
-              mobile={{ col: "fill" }}
-              tablet={{ col: 6 }}
-            >
-              <img src={graphic.image} alt={graphic.imageDescription} />
-              <h2>{graphic.heading}</h2>
-              <p>{graphic.text}</p>
+            <Grid mobile={{ col: "fill" }} tablet={{ col: 6 }}>
+              {<img src={graphic.image} alt={graphic.imageDescription} />}
```

Comment:
You don't need surrounding curly braces here. here.

Reply:
Forgot to add my graphic.image && logic there